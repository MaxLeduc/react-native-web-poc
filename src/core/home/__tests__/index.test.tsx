import React from 'react'
import { create } from 'react-test-renderer'
import { mount } from 'enzyme'
import { Text } from 'react-native'

import Home from '../index'

describe('Home', () => {
  it('renders without crashing', () => {
    const root = create(<Home />)

    expect(root.toJSON()).toMatchSnapshot()
  })

  it('renders a greeting', () => {
    const component = mount(<Home />)
    const greeting = component.find(Text)

    expect(greeting.text()).toEqual('Welcome to the future.')
  })
})
