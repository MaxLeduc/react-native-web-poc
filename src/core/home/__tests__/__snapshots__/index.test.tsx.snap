// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without crashing 1`] = `
<div
  className="css-view-1dbjc4n r-height-zh076v"
>
  <div
    className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-justifyContent-1777fci"
  >
    <div
      className="css-text-901oao r-color-howw7u"
      dir="auto"
    >
      Welcome to the future.
    </div>
  </div>
</div>
`;
